### PR TITLE
Initialized the snake and made a move function

### LINKED ISSUE #72: Initialize game
The first thing we should work on to make the game is the initialization function, named {init_game}. It should contain the initialization of the snake, the scenes, the snake snacks, the amount of energy and energy decay function call.

### LINKED ISSUE #73: Make snake's movement
Apart from initializing the game itself, the next step is to start working on the function that performs the snake's movement and assigns the WASD keys to its respective movements.

### PR DESCRIPTION
**Pull request by artP2**
This PR proposes the following changes:
- Initialize the snake in the init_game function.
- Create a simple move function.
- Assign WASD keys to a change in direction.

Partially fixes #72 and #73.

### CODE REVIEW

**Review by pedgrando — APPROVED** [review]

**Review by pedgrando — APPROVED** [review]

**Comment by guynamedped** [comment]
This PR solves partially the problems presented in issues #72 and #73.

**Review by pedgrando — APPROVED** [review]

**Comment by lucasvianav** [comment]
> This PR solves partially the problems presented in issues #72 and #73.

This should be added to the PR's description, so it'll be linked to the issues.

File: src/ttsnake.c [hunk 1]
```diff
@@ -264,12 +264,53 @@ void showscene (scene_t* scene, int number, int menu)
...
 
+/* This function moves the snake */
+
+void move_snake(){
+	int i;
+	for(i = snake.length - 1; i >= 0; i--){
+		if(i){
+			snake.positions[i].x = snake.positions[i - 1].x;
+			snake.positions[i].y = snake.positions[i - 1].y;
+		}else{
```
Comment (lucasvianav) [inline src/ttsnake.c]: Missing space
```suggestion
		} else{
```

File: AUTHORS.md [hunk 2]
```diff
@@ -18,17 +18,17 @@ the following authors:
...
-        <a href="https://github.com/Dauboau">
-            <img src="https://avatars.githubusercontent.com/u/86164187?v=4" width="100;" alt="Dauboau"/>
+        <a href="https://github.com/Float07">
+            <img src="https://avatars.githubusercontent.com/u/39651883?v=4" width="100;" alt="Float07"/>
             <br />
-            <sub><b>Daniel Contente Romanzini</b></sub>
+            <sub><b>Thiago Daniel Cagnoni De Pauli</b></sub>
         </a>
     </td>
     <td align="center">
```
Comment (lucasvianav) [inline AUTHORS.md:34]: It's better to not manually edit this file for we have an automated workflow to update it since #37

File: src/ttsnake.c [hunk 3]
```diff
@@ -333,28 +374,33 @@ void playgame (scene_t* scene)
...
+				break;
+			case 'd':
+				snake.direction = right;
+				break;
+			case 'q':
+			kill (0, SIGINT);
+			break;
+		default:
+			break;
+		}
```
Comment (lucasvianav) [inline src/ttsnake.c]: Inconsistent indentation and `default` case is not needed
```suggestion
		switch(c){
			case 'w':
				snake.direction = up;
				break;
			case 'a':
				snake.direction = left;
				break;
			case 's':
				snake.direction = down;
				break;
			case 'd':
				snake.direction = right;
				break;
			case 'q':
				kill (0, SIGINT);
				break;
		}
```

File: src/ttsnake.c [hunk 4]
```diff
@@ -333,28 +374,33 @@ void playgame (scene_t* scene)
...
 	}
-      
-    }
-  return NULL;
-
+	return NULL;
```
Comment (lucasvianav) [inline src/ttsnake.c:400]: I think it's good to add a blank line before this

**Review by lucasvianav — DISMISSED** [review]

File: AUTHORS.md (hunk 2 above)
Comment (artP2) [reply AUTHORS.md:34]: It was a github action, not mine...

Comment (artP2) [reply AUTHORS.md:34]: Any clue in how to remove this?

**Comment by pedgrando** [comment]
> > This PR solves partially the problems presented in issues #72 and #73.
> 
> This should be added to the PR's description, so it'll be linked to the issues.

Sure, man. Thanks for helping.

File: src/ttsnake.c (hunk 4 above)
Comment (artP2) [reply src/ttsnake.c:400]: ok

File: AUTHORS.md (hunk 2 above)
Comment (pedgrando) [reply AUTHORS.md:34]: I guess it just changed the order of the profiles. As it's an github automatic action, perhaps it isn't that bad.

**Review by ms-erik — APPROVED** [review]

**Pull request merged**